static/src/javascripts/projects/common/modules/video/more-in-series-container.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import Component from 'common/modules/component';
+import mediator from 'lib/mediator';
+import { checkElemsForVideos } from 'common/modules/atoms/youtube';
+
+const moreInSeriesContainerInit = (
+    el: HTMLElement,
+    mediaType: string,
+    section: string,
+    shortUrl: string,
+    series: string
+): void => {
+    const component = new Component();
+    const endpoint = `/${mediaType}/section/${section}${series
```

Comment:
Good catch! I will turn it into a [maybe string](https://flow.org/en/docs/types/maybe/).